next.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  async redirects() {
+    return [
+      {
+        source: '/open-project/:repository/:file',
```

Comment:
I wonder if we can keep this shorter, e.g.:

```suggestion
        source: '/_/:repository/:file',
```
or
```suggestion
        source: '/#/:repository/:file',
```

Reply:
@christian-bromann  I had to remove this approach, after realizing that vercel for some reason is removing the last / from vscode protocol part. An API approach works.